dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/InterpretationController.java

```diff
@@ -342,10 +407,16 @@ public void updateComment( @PathVariable( "uid" ) String uid, @PathVariable( "cu
                 }
 
                 comment.setText( content );
+                Set<User> users = interpretationService.getMentionedUsers( content ); 
+                comment.setMentions(  users );
+                interpretationService.updateSharingForMentions(interpretation, users);
+                interpretationService.updateInterpretation( interpretation );
+                interpretationService.sendNotifications(interpretation, comment, users);
+                
             }
         }
 
-        interpretationService.updateInterpretation( interpretation );
+//        interpretationService.updateInterpretation( interpretation );
```

Comment:
Minor: Can we remov the line altogether?

Reply:
Removed!